uppbox/MakePosixInstall/scripts.cpp

```diff
@@ -14,11 +14,12 @@ uname=`uname`
 if [[ "$uname" == 'Darwin' ]]; then
   DEP=""
   if which brew; then
-    echo "Homebrew is already installed"
+    echo "Homebrew is installed"
   else
-    echo "Installing Homebrew package management system (to obtain openssl libraries required for U++)"
+    echo "Homebrew is not installed"
+    echo "We use Homebrew to obtain OpenSSL libraries and clang-format required for U++"
+    echo "Follow the instructions at https://brew.sh/ to install Homebrew"
     AskContinue
```

Comment:
We should exit instead of AskContinue. You had to install brew prior to launching this script, so I would just replace this line with exit 1 or something like that.